src/main/java/core/mate/academy/model/Truck.java

```diff
@@ -5,7 +5,14 @@
  * Do not remove no-field constructor
  */
 public class Truck extends Machine {
```

Comment:
The class Truck should not start with an empty line. Please remove the empty lines before the class declaration.